service/src/main/kotlin/no/nav/su/se/bakover/service/utbetaling/UtbetalingServiceImpl.kt

```diff
@@ -91,11 +92,13 @@ internal class UtbetalingServiceImpl(
         attestant: NavIdentBruker,
         beregning: Beregning,
         simulering: Simulering,
+        uføregrunnlag: List<Grunnlag.Uføregrunnlag>
```

Comment:
Det er ven en mulighet her for at uføregraden vil føre til at vi fort stopper opp `harEndringerIUtbetalingSidenSaksbehandlersSimulering` - dette som følge av at vi ikke sender med uføregrad ved simulering av utbetalingene første gangen. Spørsmålet blir vel om vi også burde simulere med uføregrad, eller om vi skal "frita" uføregrad fra denne sjekken. Usikker på om det kanskje vil være relevant med nøyaktige verdier for skatt når det kommer til tilbakekreving (overser den i andre tilfeller), eller om vi aldri trenger å bry oss om slikt.

Reply:
Jeg sjekket med fag, og det ser ikke ut til at vi trenger uføregraden til noe annet enn utbetalingene. Vil mene at dette sies med litt usikkerhet.

Reply:
@jacob-meidel Dette er vel mer en teknisk avgjørelse. I teorien så kan vel en utbetaling bli avvist pgr. innholdet i feltet? Dvs. dersom det ikke finnes noen sjekker på det nå, betyr ikke det at det ikke kan komme noen vi ikke får med oss.

Reply:
Det eneste stedet i simuleringsapiet jeg finner `uforeGrad` er i `BeregningStoppnivaaDetaljer` :shrug:

Så det kan virke som vi ikke har mulighet til å sende dette i simuleringen. Da må du i så fall gjøre de tiltakene Jacob nevner.

Reply:
Er for så vidt enig i det overordnede poenget at det er best å simulere med de verdiene vi har tenkt å sende over - det er på mange måter også poenget. Vi vil vel avvise simuleringen i denne sjekken dersom "saksbehandlers" simulering divergerer fra "kontrollsimulering" - denne sjekken inkluderer også feltet uføregrad. Dette byr på problemer om vi ikke er konsekvent i om vi sender med dette feltet eller ikke.

Reply:
> Det eneste stedet i simuleringsapiet jeg finner `uforeGrad` er i `BeregningStoppnivaaDetaljer` 🤷
> 
> Så det kan virke som vi ikke har mulighet til å sende dette i simuleringen. Da må du i så fall gjøre de tiltakene Jacob nevner.

Skal kunne legge `grad` `SimuleringRequestBuilder.nyLinje()`

Reply:
Da skal vel denne være helt ok?